src/const.js

```diff
@@ -87,6 +93,22 @@ export const LOW_LEVEL_MODULES = [
   'system/events', 'sdk/system/events',
 ];
 
+export const DEPRECATED_ENTITIES = [
+  {
```

Comment:
Hmm. that's annoying. Can you provide a paste or commit it like that so we can see the error + affected rules?